docs/Reference/API-Methods.md

```diff
@@ -796,9 +880,9 @@ Returns the accounts for a specified block.
 
 #### Parameters
 
-* `blockHashOrNumber`: *string* - block hash or number
+* `blockHashOrNumber`: *string* - block hash or number at which to retrieve account information
 
-* `txIndex`: *number* - transaction index from which to start
+* `txIndex`: *number* - transaction index at which to retrieve account information
 
 * `address`: *string* - address hash from which to start
```

Comment:
Should these match the parameter descriptions for `txIndex` and `address` for `debug_accountAt`?